common.sh

```diff
@@ -107,9 +107,19 @@ create_instance()
     elif [ $BUILD_GDR -eq 1 ]; then
         instance_type=p4d.24xlarge
         network_interface="[{\"DeviceIndex\":0,\"DeleteOnTermination\":true,\"InterfaceType\":\"efa\",\"Groups\":[\"${slave_security_group}\"]"
-    else
-        instance_type=a1.4xlarge
-        network_interface="[{\"DeviceIndex\":0,\"DeleteOnTermination\":true,\"Groups\":[\"${slave_security_group}\"]"
+    elif [ $ami_arch = "aarch64" ]; then
+        case "${PROVIDER}" in
+            efa)
+                instance_type=c6gn.16xlarge
+                network_interface="[{\"DeviceIndex\":0,\"DeleteOnTermination\":true,\"InterfaceType\":\"efa\",\"Groups\":[\"${slave_security_group}\"]"
+                ;;
+            tcp)
```

Comment:
Might be better to leave a comment here, saying it's for rhel/centos7

Reply:
Hmmm, I thought the libfabric-ci-scripts should be as general as it is for any instance/provider test. The comment for rhel/centos 7 should be handled in the caller scripts ( and we already did)?

Reply:
I see your point. Ok, ignore my comment then.